Doppler.HtmlEditorApi/appsettings.json

```diff
@@ -19,5 +19,42 @@
   "DopplerDataBaseSettings": {
     "ConnectionString": "Server=REPLACE_FOR_SQL_SERVER;Database=REPLACE_FOR_DATABASE_NAME;User Id=REPLACE_FOR_DATABASE_USERNAME; MultipleActiveResultSets=True;",
     "Password": "REPLACE_FOR_DATABASE_PASSWORD"
+  },
+  "fields": {
+    "aliases": [
+      {
+        "canonicalName": "BIRTHDAY",
+        "aliases": [
+          "CUMPLEANOS",
+          "CUMPLEAÑOS",
+          "DATE OF BIRTH",
+          "DOB",
+          "FECHA DE NACIMIENTO",
+          "NACIMIENTO"
+        ]
+      },
+      { "canonicalName": "COUNTRY", "aliases": ["PAIS", "PAÍS"] },
+      {
+        "canonicalName": "EMAIL",
+        "aliases": [
+          "CORREO",
+          "CORREO ELECTRONICO",
+          "CORREO ELECTRÓNICO",
+          "CORREO_ELECTRONICO",
+          "CORREO_ELECTRÓNICO",
+          "E-MAIL",
+          "MAIL"
+        ]
```

Comment:
```suggestion
          "E MAIL",
          "CASILLA DE CORREO",
          "DIRECCION DE EMAIL"
          ]
```

Reply:
Maybe it is a good idea, but I am making honor to current doppler behavior, you can see more details in this conversation: https://github.com/FromDoppler/doppler-html-editor-api/pull/55#discussion_r820684090